library/Icinga/Web/Widget/Dashboard.php

```diff
@@ -77,15 +77,15 @@ public function load()
         foreach ($navigation as $dashboardPane) {
             /** @var DashboardPane $dashboardPane */
             $pane = new Pane($dashboardPane->getLabel());
```

Comment:
My quick fix for this also included the following here:
```
$pane = (new Pane($dashboardPane->getName()))->setTitle($dashboardPane->getLabel());
```
Maybe this helps? With this change only, panes won't get translated upon saving.

Reply:
Tried it, doesn't affect dashlets though. And honestly, my solution works. Maybe there's a simpler one, but any additional second used to look at this is a wasted one.